### PR TITLE
Swapped chatRoom entity to match entity deleted unnecessary code

### PR DESCRIPTION
**Pull request by joelruettimann**

### CODE REVIEW

File: src/main/java/ch/uzh/ifi/hase/soprafs21/repository/ItemRepository.java [hunk 1]
```diff
@@ -8,6 +8,6 @@
 
 @Repository("iemRepository")
```
Comment (messias-des-coden) [inline src/main/java/ch/uzh/ifi/hase/soprafs21/repository/ItemRepository.java]: Do you mean "itemRepository"?

File: src/main/java/ch/uzh/ifi/hase/soprafs21/service/MatchService.java [hunk 2]
```diff
@@ -41,12 +43,10 @@ public void createMatch(long itemIdOne, long itemIdTwo){
...
-        List<Matches> matchesByIdOne = matchRepository.findByItemIdOne(itemId);
-        List<Matches> matchesByIdTwo = matchRepository.findByItemIdTwo(itemId);
-        List<Matches> allMatchesById = new ArrayList<>();
-        allMatchesById.addAll(matchesByIdOne);
-        allMatchesById.addAll(matchesByIdTwo);
-        return allMatchesById;
+        List<Matches> fromIdOne = matchRepository.findByItemIdOne(itemId);
+        List<Matches> fromIdTwo = matchRepository.findByItemIdTwo(itemId);
+        return Stream.concat(fromIdOne.stream(), fromIdTwo.stream())
+                .collect(Collectors.toList());
```
Comment (messias-des-coden) [inline src/main/java/ch/uzh/ifi/hase/soprafs21/service/MatchService.java:49]: Does this make a difference?

Comment (joelruettimann) [reply src/main/java/ch/uzh/ifi/hase/soprafs21/service/MatchService.java:49]: jup, much faster

File: src/main/java/ch/uzh/ifi/hase/soprafs21/repository/ItemRepository.java (hunk 1 above)
Comment (joelruettimann) [reply src/main/java/ch/uzh/ifi/hase/soprafs21/repository/ItemRepository.java]: yessir

**Review by botmanski — APPROVED** [review]
already reviewed by mauro

**Pull request merged**